ScoreSaber/Core/ReplaySystem/UI/DesktopMainImberPanelView.cs

```diff
@@ -153,25 +155,28 @@ public void UpClick() {
 
 
         private void RecursivelyDisableItalics(GameObject obj) {
```

Comment:
please also rename the methods now that they are no longer recursive. Maybe something like `DisableAllItalics`

Reply:
Fixed